packages/babel-preset-stage-0/README.md

```diff
@@ -1,19 +1,58 @@
 # @babel/preset-stage-0
 
-> Babel preset for stage 0 plugins
+> As of v7.0.0-beta.52, we've decided to remove
+the official Babel Stage presets. You can find more information
+at issue [#7770](https://github.com/babel/babel/issues/7770), but
+the TL;DR is that it's causing more harm than convenience in that
+the preset is always out of date, each change is usually going to
+require a major version bump and thus people will be behind,
+and it encouraged too many people to opt-in to too many proposals
+that they didn't intend to. This is intended to be the last publish
+of "@babel/preset-stage-0"
 
-See our website [@babel/preset-stage-0](https://babeljs.io/docs/en/next/babel-preset-stage-0.html) for more information.
+---
 
-## Install
+If you want the same configuration as before:
 
-Using npm:
+```json
+{
+  "plugins": [
+    // Stage 0
+    "@babel/plugin-proposal-function-bind",
 
-```sh
-npm install --save-dev @babel/preset-stage-0
+    // Stage 1
+    "@babel/plugin-proposal-export-default-from",
+    "@babel/plugin-proposal-logical-assignment-operators",
+    ["@babel/plugin-proposal-optional-chaining", { "loose": false }],
+    ["@babel/plugin-proposal-pipeline-operator", { "proposal": "minimal" }],
+    ["@babel/plugin-proposal-nullish-coalescing-operator", { "loose": false }],
+    "@babel/plugin-proposal-do-expressions",
+
+    // Stage 2
+    ["@babel/plugin-proposal-decorators", { "legacy": true }],
+    "@babel/plugin-proposal-function-sent",
+    "@babel/plugin-proposal-export-namespace-from",
+    "@babel/plugin-proposal-numeric-separator",
+    "@babel/plugin-proposal-throw-expressions",
+
+    // Stage 3
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-syntax-import-meta",
+    ["@babel/plugin-proposal-class-properties", { "loose": false }],
+    "@babel/plugin-proposal-json-strings"
+  ]
+}
 ```
 
-or using yarn:
+We recommend that make your own presets to use across projects for
```

Comment:
Is this something we want to recommend? I'm not sure how I feel about this being in here as a recommendation for good.

Reply:
Hmm yeah, I guess it shouldn't have to be a recommendation but more that people tend to forget you can make your own

Reply:
Yeah, I'm fine if we want to say that creating a custom preset is an option, I think the wording right now just makes it sound like what we're expecting everyone to do, which isn't good.

Maybe something like

> If you're using the same configuration across many separate projects, keep in mind that you can also create your own custom presets with whichever plugins and presets you're looking to use.